Source/WebCore/page/RemoteFrameView.h

```diff
@@ -26,19 +26,17 @@
 #pragma once
 
 #include "FrameView.h"
+#include "RemoteFrame.h"
```

Comment:
This shouldn't be necessary.  You can put the definition of frame() into the cpp file.